### DISCUSSION TITLE
How to Install on win 11

### DISCUSSION THREAD

**Discussion by tismofied** [Romanitho/Winget-AutoUpdate#727] [Q&A] [answered]
I am embarrassed to ask for a how to install. which asset do I download the counter zip?
also, where do I place the folder? or this app have an installer?

**Comment by Romanitho** [comment, marked as answer]
You can use the msi
https://github.com/Romanitho/Winget-AutoUpdate/releases/tag/v1.22.2-n

**Reply by tismofied** [reply]
thank you!!!

**Reply by Romanitho** [reply]
Still in beta, but the stable release is arriving soon.
